includes/definitions/discovery/aloha.yaml

```diff
@@ -0,0 +1,48 @@
+mib: EXCELIANCE-MIB
+modules:
+    os:
+        hardware: EXCELIANCE-MIB::alHardwareModel
+        serial: EXCELIANCE-MIB::alHardwareETHID
+        version: EXCELIANCE-MIB::alProductSubVersion
+
+    sensors:
+        state:
+            data:
+                -
+                    oid: alFrontendTable
+                    value: alFrontendStatus
+                    num_oid: '.1.3.6.1.4.1.23263.4.2.1.3.2.1.13.{{ $index }}'
+                    index: 'alFrontendStatus.{{ $index }}'
+                    descr: alFrontendName
+                    state_name: alFrontendStatus
+                    group: "Frontend status"
+                    states:
+                        - { value: 0, generic: 0, graph: 1, descr: no }
+                        - { value: 1, generic: 0, graph: 1, descr: OPEN }
+
+                -
+                    oid: alBackendTable
+                    value: alBackendStatus
+                    num_oid: '.1.3.6.1.4.1.23263.4.2.1.3.3.1.20.{{ $index }}'
+                    index: 'alBackendStatus.{{ $index }}'
+                    descr: alBackendName
+                    group: "Backend status"
+                    state_name: alBackendStatus
+                    states:
+                        - { value: 2, generic: 1, graph: 1, descr: DOWN }
+                        - { value: 1, generic: 0, graph: 1, descr: UP }
+
+                -
+                    oid: alServerTable
+                    value: alServerStatus
+                    num_oid: '.1.3.6.1.4.1.23263.4.2.1.3.4.1.19.{{ $index }}'
+                    index: 'alServerStatus.{{ $index }}'
+                    descr: alServerName
+                    #group: "{{ $alBackendTable }}.{{ $alServerBackendID }}"
```

Comment:
I would like this to work, but its not possible at the moment.
If i pre-cache it it returns an array like;
```
alBackendName.1.2 = "stats"
alBackendName.1.17 = "api-library-search"
```
Problem is the index sent to the "translation" function is for example `1.17.4`.
`alServerBackendID` contains `17` which would be nice to use (since that's how you *should* map it)